### PR TITLE
Remove mentions of the FPS chart in the Performance docs

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
See the [rendered section](https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/?branch=pr-en-us-2242#analyze-frames-per-second) for review.

This PR removes mentions of the FPS chart from the Application tool since this chart does not exist anymore.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 1]
```diff
@@ -114,13 +114,11 @@ The main metric for measuring the performance of any animation is frames per sec
 
 1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
 
-   ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:117]: Can you explain why you removed this image? The text before says "look at the FPS chart below", but there isn't a chart below anymore. So, it's a bit confusing. Shouldn't we just keep this image?

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 2]
```diff
@@ -114,13 +114,11 @@ The main metric for measuring the performance of any animation is frames per sec
...
+1. The **CPU** chart is displayed.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The fact that the **CPU** chart is full of color means that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
 
    ![The CPU chart and Summary panel.](../media/evaluate-performance-performance-cpu-chart.msft.png)
 
-1. Hover over the **FPS**, **CPU**, or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of animations.
+1. Hover over the **CPU**, or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of animations.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:121]: Why are we removing FPS from the first sentence? Does hovering over the FPS chart not do anything?

**Review by captainbrosset — DISMISSED** [review]

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md (hunk 2 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:121]: I don't know the reason why. It was the work item. Shall I ask the SMEs? https://microsoft.visualstudio.com/Edge/_queries/edit/41648598/?triage=true

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md (hunk 1 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:117]: From Reeva:  [10:06 AM] Reeza Ali
[9:47 AM] Reeza Ali
Jason Andrew (Wipro Designit Services, Inc.)If that's the case the work item needs to be revise and everything redone. I was just worker bee doing it, which is how it came up, but I can revise. Reeza Ali? Shall I do that?
Thanks for asking for clarification. 
 
I met with Zoher (1 month ago), and then Zoher and Rachel about 1-2 weeks ago. This was the guidance and changes they recommended we make to the doc. 
 
In terms of next steps: Let's button it up and send it to them for review. They can then guide us on what might need to change or not. 
 
[9:48 AM] Reeza Ali
Also, Rachel is planning to overhaul the performance docs in Dec.

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 3]
```diff
@@ -114,13 +114,11 @@ The main metric for measuring the performance of any animation is frames per sec
 
 1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
```
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:116]:
```suggestion
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 4]
```diff
@@ -114,13 +114,11 @@ The main metric for measuring the performance of any animation is frames per sec
...
 
-   ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)
-
-1. Below the **FPS** chart, the **CPU** chart is displayed.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The fact that the **CPU** chart is full of color means that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
+1. The **CPU** chart is displayed.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The fact that the **CPU** chart is full of color means that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
```
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:117]:
```suggestion
1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
```

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md (hunk 2 above)
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:121]: FPS chart does not exist.

Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:121]:
```suggestion
1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.
```

**Review by hxlnt — APPROVED** [review]
In addition to the comments left here, all screenshots should be replaced because the FPS chart no longer exists.

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 5]
```diff
@@ -112,15 +112,11 @@ Once you have a recording of the page's performance, you can assess the page's p
 
 The main metric for measuring the performance of any animation is frames per second (FPS).  Users are happy when animations run at 60 FPS.
 
-1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
-
-   ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:117]: Before we merge this, can you check if this image is used anywhere else in the docs? If not, then please delete the image file. Thanks!

**Review by captainbrosset — APPROVED** [review]
Approved, with just one comment. Please take a look.

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 6]
```diff
@@ -112,15 +112,11 @@ Once you have a recording of the page's performance, you can assess the page's p
 
 The main metric for measuring the performance of any animation is frames per second (FPS).  Users are happy when animations run at 60 FPS.
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:113]: I think this should be cut and the `### Analyze frames per second` section title. We don't have FPS tracked any more in the Performance tool so everything related to FPS needs to be removed

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md [hunk 7]
```diff
@@ -112,15 +112,11 @@ Once you have a recording of the page's performance, you can assess the page's p
...
 
    ![The CPU chart and Summary panel.](../media/evaluate-performance-performance-cpu-chart.msft.png)
 
-1. Hover over the **FPS**, **CPU**, or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of animations.
+1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.
 
    ![View a screenshot of the page around the 2500ms mark of the recording.](../media/evaluate-performance-performance-screenshot-hover.msft.png)
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:122]: I think the next step also needs to be removed. The **Frames** section no longer displays FPS for each particular frame. I can't reproduce the screenshot shown here.

File: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md (hunk 6 above)
Comment (JasonAndrewWriter) [reply microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md:113]: done

**Review by zoherghadyali — APPROVED** [review]

**Pull request merged**